GARfVDB (#308)
openvdb/fvdb-reality-capture: docs/api/frgs/mesh_basic.rst

   │ -o PATH, --output-path PATH                                                                    │
   │                         Path to save the extracted mesh (default is "mesh.ply"). (default:     │
   │                         mesh.ply)                                                              │

>    │ -d STR, --device STR    Extract a mesh from a Gaussian Splat reconstruction. (default: cuda:0) │

Reviewer: <!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**💡 suggestion:** ‏

```suggestion
   │ -d STR, --device STR    Device to use for computation (default is "cuda:0"). (default: cuda:0) │
```
